### DISCUSSION TITLE
element click intercepted

### DISCUSSION THREAD

**Discussion by wangli-wangli** [webdriverio/webdriverio#7274] [Q&A]
![image](https://user-images.githubusercontent.com/33982322/129303356-0bee3105-9435-4a46-9ade-7dc18a71ea06.png)
element click intercepted:
Click on an element, but it is accepted by an element in the same position on another layer.
I've tried:
![image](https://user-images.githubusercontent.com/33982322/129303522-7c206917-9884-4975-ac81-090774327263.png)
However, executeScript parameters cannot be Elements
What can I do

**Comment by christian-bromann** [comment]
You are trying to click on an element that is covered by a different element. This would not work in real life either. I recommend to investigate which element is covering your target and re-evaluate what automation you want to do.
